build.js

```
@@ -0,0 +1,77 @@
+  const name = path.basename(file, '.js');
+  // Remove module.exports assignment
+  let content = rawContent.replace(/module\.exports\s*=\s*\{[^}]*\};?/, '');
+  // Rename main to module name
+  content = content.replace(/function\s+main/, `function ${name}`);
+  // Replace requires to other core modules with component references
+  content = content.replace(/require\(\s*['"]\.\/([\w-]+)['"]\s*\)/g, "components['$1']");
+  // Wrap module code and assign exports
+  output += `
+  // component: ${name}
+  (function() {
+${content.split('\n').map(line => '    ' + line).join('\n')}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)